### ISSUE TITLE
Wrong handling of Google URLs

### ISSUE THREAD

**Issue by norskpost** [organicmaps/organicmaps#3472]
Android
Have problem opening direction link from Firefox browser, open link from Bing search works only once, then need to restart "om" app, links from duckduckgo and google searches does hardly open.
Example:
Not work.......

geo:?q=Thorvald Meyers gate 30B, Oslo, NO 0555

...........

https://maps.google.com/maps?client=firefox-b-m&channel=ts&um=1&ie=UTF-8&fb=1&gl=no&sa=X&geocode=KScqCg7_XEFGMfpG0WL1sDvO&daddr=XL-bygg+Ski,+Kjeppestadveien+44,+1400+Ski&ved=2ahUKEwiH1cbplqv6AhUR-6QKHejIB3gQlDt6BAgMEAQ

Work,..........

https://www.google.com/maps/dir/My+location/XL-Bygg,Kjeppestadveien+44%2c+1400+Ski/@59.71269989013672,10.857339859008789,8z/

**Comment by biodranik** [comment]
Here is the problem in how we parse google URLs:
```
09-23 14:54:52.485 (main) I/DownloadResourcesLegacyActivity: processIntent(): Incoming intent uri: Intent { act=android.intent.action.VIEW cat=[android.intent.category.BROWSABLE] dat=https://www.google.com/... flg=0x10000000 cmp=app.organicmaps/com.mapswithme.maps.SplashActivity (has extras) }
```

**Comment by rtsisyk** [comment]
Duplicate of https://github.com/organicmaps/organicmaps/issues/320. Please subscribe to #320.
